.storybook/wcagConfig.ts

```diff
@@ -0,0 +1,6 @@
+import { axeOptionsConfig } from 'axe-playwright';
```

Comment:
Does this need to be a separate file? If it's only ever going to be used in test-runner.ts we can put them together

Reply:
yen preferred it to be separate so it can be used in both test-runner.ts and preview.js